e2e/wdio/platform/tests/input-group.e2e-spec.ts

```diff
@@ -0,0 +1,143 @@
+    });
+
+    it('have right text addon and accept values', () => {
+        webDriver.waitForDisplayed(inputGroupPage.rightAlignedTextInputTextAddon);
+        webDriver.setValue(inputGroupPage.rightAlignedTextInput, inputGroupTestData.numeric_value);
+
+        expect(webDriver.getText(inputGroupPage.rightAlignedTextInputTextAddon)).toContain('0.00');
+        expect(webDriver.getValue(inputGroupPage.rightAlignedTextInput)).toBe(inputGroupTestData.numeric_value);
+    });
+
+    it('have right and left text addons and accept values', () => {
+        webDriver.waitForDisplayed(inputGroupPage.rightLeftAlignedTextInputTextAddon);
+        webDriver.setValue(inputGroupPage.leftAndRightAlignedTextInput, inputGroupTestData.numeric_value);
+
+        expect(webDriver.getText(inputGroupPage.rightLeftAlignedTextInputTextAddon)).toContain('$');
+        expect(webDriver.getText(inputGroupPage.rightLeftAlignedTextInputTextAddon, 1)).toContain('0.00');
+        expect(webDriver.getValue(inputGroupPage.leftAndRightAlignedTextInput)).toBe(inputGroupTestData.numeric_value);
+    });
+
+    it('have button, text addons and accept values', () => {
+        webDriver.waitForDisplayed(inputGroupPage.buttonInputSubmitButton);
+        webDriver.setValue(inputGroupPage.buttonInput, inputGroupTestData.numeric_value);
+
+        // Check if clickable. No logic behind the click
+        expect(webDriver.getText(inputGroupPage.buttonInputLeftAndRightTextAddon)).toContain('$');
+        expect(webDriver.getText(inputGroupPage.buttonInputLeftAndRightTextAddon, 1)).toContain('0.00');
+        expect(webDriver.isElementClickable(inputGroupPage.buttonInputSubmitButton)).toBe(true);
+        expect(webDriver.getValue(inputGroupPage.buttonInput)).toBe(inputGroupTestData.numeric_value);
+    });
+
+    it('have icon addon and accept values', () => {
+        webDriver.waitForDisplayed(inputGroupPage.iconInputEmailIcon);
+        webDriver.setValue(inputGroupPage.iconInput, inputGroupTestData.email_value);
+
+        expect(webDriver.getValue(inputGroupPage.iconInput)).toBe(inputGroupTestData.email_value);
+    });
+
+    it('compact have button and text addon', () => {
+        webDriver.waitForDisplayed(inputGroupPage.compactGroupInput);
+        webDriver.setValue(inputGroupPage.compactGroupInput, inputGroupTestData.string_value);
+
+        expect(webDriver.getText(inputGroupPage.compactGroupLeftTextAddon)).toContain('$');
+        expect(webDriver.getText(inputGroupPage.compactGroupButtonAddon)).toContain('Submit');
+        expect(webDriver.isElementClickable(inputGroupPage.compactGroupButtonAddon)).toBe(true);
+        expect(webDriver.getValue(inputGroupPage.compactGroupInput)).toBe(inputGroupTestData.string_value);
+    });
+
+    it('compact be smaller than the default', () => {
+        const defaultHeight = webDriver.getElementSize(inputGroupPage.leftAlignedTextInput) as WebdriverIO.SizeReturn;
```

Comment:
shouldn't be `WebdriverIO.SizeReturn` a return type of `getElementSize`

Reply:
Done.